assets/javascripts/discourse/widgets/chat-state-post-small-action.js

```diff
@@ -0,0 +1,35 @@
+import RawHtml from "discourse/widgets/raw-html";
+import { userPath } from "discourse/lib/url";
+import { autoUpdatingRelativeAge } from "discourse/lib/formatter";
+import I18n from "I18n";
+import { createWidget } from "discourse/widgets/widget";
+
+export default createWidget("chat-state-post-small-action", {
+  click(event) {
+    if (event.target.classList.contains("open-chat")) {
+      event.preventDefault();
+
+      const topicController = this.container.lookup("controller:topic");
+      const chatChannel = topicController.model.chat_channel;
+
+      if (chatChannel) {
+        this.appEvents.trigger("chat:open-channel-for-chatable", chatChannel);
```

Comment:
note @markvanlan I found this was slightly unexpected to have something called "open-channel-FOR-CHATABLE" and having it expecting a chat channel